### DISCUSSION TITLE
recipes/desserts/cooks-illustrated-lemon-bars/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#969] [Comments]
# recipes/desserts/cooks-illustrated-lemon-bars/

A collection of my recipes.

https://nicholaswilde.io/recipes/desserts/cooks-illustrated-lemon-bars/

**Comment by nicholaswilde** [comment]
Move to cookies and bars folder.

**Reply by nicholaswilde** [reply]
Try freezing the crust dough for 5-10 minutes before baking.
Dock crust before baking.

**Reply by nicholaswilde** [reply]
Blanch eggs and sugar first until smooth.
